lib/LANraragi/Model/Tankoubon.pm

```diff
@@ -250,6 +250,19 @@ sub add_to_tankoubon( $tank_id, $arc_id ) {
     return ( 0, $err );
 }
 
+# get_tankoubons_file(arcid)
+#   Gets a list of Tankoubons where archive ID is contained.
+#   Returns an array of tank IDs.
+sub get_tankoubons_file($arcid) {
+
+    my $logger = get_logger( "Tankoubon", "lanraragi" );
```

Comment:
this sub can probably be trimmed down to just `return get_tankoubons_by_file($arcid);`